README.md

```diff
@@ -103,6 +107,14 @@ the project:
 ``
 
 and then run the project with IOS emulator.
+---
+
+Some version of Gradle create unnecessary files that may result in errors while building the app. Gradle 2.14.1 has proven to work without any issues.
+If errors persist, make sure gradle-wrapper.properties is using the right version of Gradle.
```

Comment:
specify path to `gradle-wrapper.properties` and a little example of how to specify the wrapper?